.pre-commit-config.yaml

```diff
@@ -94,7 +94,6 @@ repos:
       - id: insert-license
         name: Add licence for all shell files
```

Comment:
```suggestion
         name: Add licence for all files
```

Reply:
It's still for all shell files @feluelle :) (see the regexp = . .sh + breeze + breeze-complete). Pre-commit has two options for matching file selection - type and regexp in file name. I removed the type and left the regexp with .sh